### PR TITLE
[12.0][MIG] website_event_questions_free_text

### PR DESCRIPTION
**Pull request by Saran440**

### CODE REVIEW

File: website_event_questions_free_text/views/event_event_views.xml [hunk 1]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (chienandalu) [inline website_event_questions_free_text/views/event_event_views.xml]: Don't drop the xml prolog

File: website_event_questions_free_text/views/event_templates.xml [hunk 2]
```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (chienandalu) [inline website_event_questions_free_text/views/event_templates.xml]: Same here

**Review by chienandalu — CHANGES_REQUESTED** [review]
Thanks @Saran440 !

Some little changes:

**Comment by Saran440** [comment]
@chienandalu Fixed it. I thought it not necessary. Thank you for your reviewed.

File: website_event_questions_free_text/readme/CONTRIBUTORS.rst [hunk 3]
```diff
@@ -1,3 +1,4 @@
 * `Tecnativa <https://www.tecnativa.com>`_:
 
-  * Pedro M. Baeza
\ No newline at end of file
+  * Pedro M. Baeza
+* Saran Lim. <saranl@ecosoft.co.th>
```
Comment (pedrobaeza) [inline website_event_questions_free_text/readme/CONTRIBUTORS.rst:5]: Extra empty line in between is needed for proper RST syntax

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
And squash all your migration commits

**Review by chienandalu — APPROVED** [review]
Tested functionally :+1:

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@sbidoul this one has been tagged as Merged by GitHub. I suppose it's due to non extra commits in the rebased branch.

**Comment by sbidoul** [comment]
@pedrobaeza Yes, it's because the PR was fresh enough to already be on top of the main branch, so the rebase did not change the commits of the PR.

**Pull request merged**